calc.py

```diff
@@ -0,0 +1,56 @@
+import math
+
+
+def add(a, b):
+    check_types(a, b)
+    return a + b
+
+
+def subtract(a, b):
+    check_types(a, b)
+    return a - b
+
+
+def multiply(a, b):
+    check_types(a, b)
+    return a * b
+
+
+def log(a):
+    check_type(a)
+    return math.log(a)
+
+
+def log10(a):
+     if a > 0:
+      check_type(a)
+      return math.log10(a)
+     else:
+      raise TypeError('input parameter must be positive number')
+
+
+def check_types(a, b):
+    check_type(a, 'First')
+    check_type(b, 'Second')
+
+
+def check_type(param, name=''):
+    if param is None:
+        raise TypeError
```

Comment:
Would it be better to add an appropriate error message while raising the TypeError?